_posts/2022-11-21-vws-rse-cop-summary.md

```diff
@@ -0,0 +1,130 @@
+---
+layout: post
+title: Encouraging and Enabling Mutual Ownership in a RSE Community of Practice
+tags: [2022 virtual workshop]
+posted_by: Miranda Mundt
+---
+
+
+_Authors: Miranda Mundt, Jonathan Bisila, Jon Guyer, Daniel Howard,
+Daniel S. Katz, Reed Milewicz, Henry Schreiner, Joshua Teves, Chris Wiswell_
+
+## Introduction
+
+The explosion of Research Software Engineers (RSEs) in the United States
+created the opportunity to form communities of practice (CoP), groups that
+share a passion for an activity and learn how to do it better as they interact
+regularly, specifically to benefit their local organization or region. CoPs,
+however, only succeed insofar as their members feel heard, included, and valued.
+
+As part of the [2022 US-RSE Virtual Workshop](https://us-rse.org/virtual-workshop-2022/),
+Jonathan Bisila, Reed Milewicz, and Miranda Mundt of Sandia National Laboratories
+moderated the breakout session “Encouraging and Enabling Mutual Ownership in a
+RSE Community of Practice.” In this session, participants from industry,
+academia, and national laboratories and agencies across the country joined
+together to create actionable suggestions for how to get members involved in a
+community of practice and how to make them feel invested and included. The authors
```

Comment:
This is probably me nitpicking, but I think I would add a sentence before "The authors..." saying "This blogposts summarizes their discussion. The authors hope that it can assist..."
Right now I assume that the blogpost is the summary of the results, but it's no clearly stated if that makes sense.